ansible/roles/atmo-vnc/tasks/realvnc.yml

```diff
@@ -1,7 +1,8 @@
 ---
 
 - name: set environment variables for legacy systems
-  set_fact: ansible_python_interpreter=/usr/bin/python
+  set_fact:
+    ansible_python_interpreter: /usr/bin/python
```

Comment:
How many times do we need this in a role? I think you also have it in `tasks/main.yml`.